### PR TITLE
build: set preid prerelease to beta

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -134,21 +134,21 @@ jobs:
...
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'beta' }}
         with:
-          tag_name: "templates-0.0.0-prerelease"
+          tag_name: "templates-0.0.0-beta"
         env:
           GITHUB_TOKEN: ${{ secrets.CD_PAT }}
 
       - name: release templates' prerelease version to github
-        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.ref == 'refs/heads/prerelease' }}
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'beta' }}
```
[the remark this answers is not in the document]
Comment (wenytang-ms) [reply .github/workflows/cd.yml:146]: no need, the validate in the beginning already do the branch policy check.

File: .github/workflows/vscode-marketplace.yml [hunk 2]
```diff
@@ -35,13 +35,13 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           workflow: cd.yml
           path: .
-
+
```
[the remark this answers is not in the document]
Comment (wenytang-ms) [reply .github/workflows/vscode-marketplace.yml:38]: thanks

File: .github/workflows/cd.yml [hunk 3]
```diff
@@ -134,21 +134,21 @@ jobs:
...
         uses: richardsimko/update-tag@v1
-        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.ref == 'refs/heads/prerelease' }}
+        if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && github.event.inputs.preid == 'beta' }}
         with:
-          tag_name: "templates-0.0.0-prerelease"
+          tag_name: "templates-0.0.0-beta"
         env:
           GITHUB_TOKEN: ${{ secrets.CD_PAT }}
 
       - name: release templates' prerelease version to github
```
[the remark this answers is not in the document]
Comment (wenytang-ms) [reply .github/workflows/cd.yml]: sure

**Review by hund030 — APPROVED** [review]

File: .github/workflows/cd.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (qinezh) [reply .github/workflows/cd.yml:146]: As 'dev' branch is allowed, what if the CD pipeline is triggered on 'dev' branch and set preid to 'beta' by accident?

[the remark this answers is not in the document]
Comment (wenytang-ms) [reply .github/workflows/cd.yml:146]: will be blocked on the validate step. https://github.com/OfficeDev/TeamsFx/blob/6f92bb6f7462ba9d0cac0c99a0b31b1f54e7a842/.github/workflows/cd.yml#L39

[the remark this answers is not in the document]
Comment (qinezh) [reply .github/workflows/cd.yml:146]: I see, thanks for explanation.

**Review by qinezh — APPROVED** [review]

**Pull request merged**